synapse/lib/stormtypes.py

```diff
@@ -953,8 +953,11 @@ def ctordata(path=None):
     async def _methNodeNdef(self):
         return self.valu.ndef
 
-    async def _methNodeRepr(self, name=None):
-        return self.valu.repr(name=name)
+    async def _methNodeRepr(self, name=None, defv=None):
+        rval = self.valu.repr(name=name)
```

Comment:
This has to catch s_exc.NoSuchProp and set rval to None if defval is not None; otherwise we should raise a StormRuntimeError here.